source/fundamentals/aggregation.txt

```diff
@@ -179,6 +179,59 @@ The above aggregation should produce the following results:
    {"_id": 4, "count": 2}
    {"_id": 5, "count": 1}
 
+Explain Aggregation Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To explain the execution plan of the operation, append the :java-docs:`explain()
+<apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`
+method containing a :java-core-api:`verbosity level <com/mongodb/ExplainVerbosity>` to the ``aggregates()`` method.
```

Comment:
Issue: I don't think it is correct to say "containing a verbosity level" to describe the verbosity level passed to the method. I think this should be rephrased to be closer to "pass the explain() method a verbosity level to determine the contents of the explanation".